### PR TITLE
Create a github issue template

### PR DESCRIPTION
**Pull request by rossjones**
Creates an issue template for Github so that we encourage users to provide more information.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+Proposed fixes:
+
+-
+-
+-
+
```
Comment (wardi) [inline .github/PULL_REQUEST_TEMPLATE.md]: do we want a checklist here? something like

```md
- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
```

Comment (rossjones) [inline .github/PULL_REQUEST_TEMPLATE.md]: Really good idea.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+### Features:
+
+- [ ] includes tests covering changes
+- [ ] includes updated documentation
+- [ ] includes user-visible changes
+- [ ] includes API changes
+- [ ] includes bugfix for possible backport
```
Comment (wardi) [inline .github/PULL_REQUEST_TEMPLATE.md]: "please [X] all the boxes that apply" just to make it clear we're not requiring all of the above

Comment (rossjones) [inline .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Pull request merged**